lib/dockerregistry/inventory.go

```diff
@@ -2329,7 +2344,11 @@ func (sc *SyncContext) GarbageCollect(
 	if customProcessRequest != nil {
 		processRequest = *customProcessRequest
 	}
-	sc.ExecRequests(populateRequests, processRequest)
+
+	errorEncountered := sc.ExecRequests(populateRequests, processRequest)
+	if errorEncountered != nil {
+		klog.Infof(errorEncountered.Error())
```

Comment:
You could use `klog.Info(err)` here directly, probably. No use in using `Infof` if you are not using any formatting string like `"%v"`.